logs/fluent-bit/k8s-helm/coralogix/values.yaml

```diff
@@ -3,10 +3,13 @@ fluent-bit:
 
   image:
     repository: coralogixrepo/coralogix-fluent-bit-multiarch
-    tag: v0.0.2
+    tag: v0.0.3
 
   serviceMonitor:
     enabled: true
+    additionalEndpoints:
+      - port: metrics
+        path: /metrics
```

Comment:
* Does The upgrade impact the existing `/api/v1/metrics/prometheus`? 
* `/api/v1/metrics/prometheus` will remain the default?

Reply:
Hi, @avivgold098 yes that path remains the same and there's no impact on the existing metrics, this is something I also discussed with the Fluent-bit upstream charts.

By default the storage metrics are disabled so the default metrics remains the `/api/v2/metrics/prometheus` exporting basic metrics